src/main/java/net/sf/jabref/gui/EntryTypeDialog.java

```diff
@@ -115,14 +110,21 @@ private JPanel createEntryGroupsPanel() {
         JPanel panel = new JPanel();
         panel.setLayout(new VerticalLayout());
 
-        if (biblatexMode) {
-            panel.add(createEntryGroupPanel("BibLateX", EntryTypes.getAllValues(bibDatabaseContext.getMode())));
+        if (frame.getCurrentBasePanel().getBibDatabaseContext().isBiblatexMode()) {
+            panel.add(createEntryGroupPanel("BibLateX", BibLatexEntryTypes.ALL));
+
+            List<EntryType> customTypes = EntryTypes.getAllCustomTypes(BibDatabaseMode.BIBLATEX);
```

Comment:
Would it be possible to extract the following code in a private method? It is duplicated below and the only difference is the mode.